ui-components/src/notifications/AlertBox.tsx

```diff
@@ -29,6 +31,8 @@ const AlertBox = (props: AlertBoxProps) => {
     colorClasses[props.type || "alert"],
     ...(props.inverted ? ["invert"] : []),
     ...(props.className ? [props.className] : []),
+    ...(props.boundToLayoutWidth ? [] : ["fullWidth"]),
+    ...(props.narrow ? ["narrow"] : []),
   ].join(" ")
```

Comment:
I'm not too keen on how this is handled, but I'm guessing it's not the only instance of this, it's already there and it's so minor.